optuna/pruners/hyperband.py

```diff
@@ -63,9 +64,11 @@ class HyperbandPruner(BasePruner):
             in the paper.
             See the details for :class:`~optuna.pruners.SuccessiveHalvingPruner`.
         max_resource:
-            A parameter for specifying the maximum resource allocated to a trial noted as :math:`R`
-            in the paper. This value represents and should match the maximum iteration steps (e.g.,
-            the number of epochs for neural networks).
+            A parameter for specifying the maximum resource allocated to a trial. :math:`R` in the
+            paper corresponds to ``max_resource / min_resource``. This value represents and should
+            match the maximum iteration steps (e.g., the number of epochs for neural networks).
+            When this argument is "auto", the maximum resource is estimated according to the
+            completed trials. The default value of this argument is "auto".
```

Comment:
With the last couple of sentences, this PR depends on https://github.com/optuna/optuna/pull/1171. Is it intentional?